src/main/java/org/apache/maven/plugins/help/EffectivePomMojo.java

```diff
@@ -82,6 +91,14 @@
     @Parameter( property = "artifact" )
     private String artifact;
 
+    /**
+     * Output POM input location as comments.
+     * 
+     * @since 3.2.0
+     */
+    @Parameter( property = "verbose", defaultValue = "false" )
+    private boolean verbose = false;
+
```

Comment:
I think 'verbose' it too generic.
What about 'outputLocations' ?

Reply:
we need something that is user friendly

Reply:
Ok